src/tuples-and-arrays/arrays/arrays.md

```diff
@@ -0,0 +1,33 @@
+# Arrays
+
+```rust,editable
```

Comment:
This code block is missing the `<!-- mdbook-xgettext: skip -->` comment that it had on the original slide. Please add it back in so that this code block isn't included in the translations.